modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -250,6 +250,7 @@ export class ResponseError extends Error {
     override name: "ResponseError" = "ResponseError";
     constructor(public response: Response, msg?: string) {
         super(msg);
+        this.response = response;
```

Comment:
Thanks to „public response“, the response already is a member property, no?

https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties

Reply:
Oh, I wasn't aware of typescript parameter properties, it looks neat, thanks!

Then yes, it should work. I'm not at my computer but I'll take a look on Monday.